### ISSUE TITLE
Paint with Words

### ISSUE THREAD

**Issue by dblunk88** [huggingface/diffusers#1999]
Just saw this https://github.com/cloneofsimo/paint-with-words-sd which is based on Nvidia's paper of paint with words. If it's not yet implemented, a community pipeline would be pretty cool.

Essentially it's img2img, but with more control as a description of what each color should be is supplied

**Comment by cybergeekgyan** [comment]
What are you expecting to do on this issue to create the community pipeline or is it just an example ?
I would like to take this issue if no-one has taken it yet @dblunk88

**Comment by dblunk88** [comment]
> What are you expecting to do on this issue to create the community pipeline or is it just an example ? I would like to take this issue if no-one has taken it yet @dblunk88

Just threw it out openly into the room, go ahead and take if you want it 🙂

**Comment by cybergeekgyan** [comment]
Great if you can assign me this issue I will start working on 👍

**Comment by pcuenca** [comment]
Awesome @cybergeekgyan, I assigned it to you, do not hesitate to ask any questions. Thanks @dblunk88 for the suggestion!

**Comment by cybergeekgyan** [comment]
yes I am working on the issue and setup my develpment environment further

**Comment by HimariO** [comment]
Hi @cybergeekgyan, I noticed that you're still working on the setup development environment. Do you mind if I create a draft PR using the code I've been recently experimenting with? I believe it could be a fantastic starting point for this issue.

**Comment by dblunk88** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. If you think this still needs to be addressed please comment on this thread.
> 
> Please note that issues that do not follow the [contributing guidelines](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md) are likely to be ignored.

bumping
